README.md

```diff
@@ -184,3 +182,5 @@ The **Statistics Module** saves some statistics of the cloud. The data is displa
 
 ### Chat + Tab
 The **Chat + Tab Module** provides **prefixes** and **suffixes** in the tablist and adds a **chat configuration**.
+<br />
+You can **download** it [here](https://repo.thesimplecloud.eu/artifactory/gradle-release-local/eu/thesimplecloud/simplecloud/simplecloud-module-chat-tab/2.2.0/simplecloud-module-chat-tab-2.2.0.jar)
```

Comment:
If you add here the download link for this module, you have to do it also to all other modules.